app/Models/Entry.php

```diff
@@ -17,10 +17,14 @@ class FreshRSS_Entry extends Minz_Model {
 	 */
 	private $guid;
 
+	/** @var string */
```

Comment:
Can't you add the type directly in the property definition?

Reply:
That would require PHP 7.4+: https://php.net/language.types.declarations

With PHP 7.2 (which we support at the moment): `PHP Parse error:  syntax error, unexpected 'string' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST) in ./app/Models/Entry.php on line 18`

Reply:
Let's come back to it in a few months https://github.com/FreshRSS/FreshRSS/discussions/3321#discussioncomment-835704

Reply:
Interesting reading; there are several details I did not know (e.g. about uninitialized properties) https://wiki.php.net/rfc/typed_properties_v2